crt_portal/cts_forms/templates/forms/report.html

```diff
@@ -33,9 +33,7 @@ <h1 class="padding-top-5 margin-0">
             </li>
           {% endfor %}
         </ol>
-        <!-- uncomment when there is more than one step on form
```

Comment:
Unfortunately the connecting line is all out of sync when we uncomment with only two steps; I was too optimistic when I wrote this code comment! Will back out the uncommenting; I think no line is better than ugly-looking line.

Reply:
sounds good